<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportForm</class>
 <widget class="QWidget" name="ExportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gb_book_options">
     <property name="title">
      <string>书籍信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_author">
        <property name="text">
         <string>作者</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_author">
        <property name="text">
         <string>XiYang6666/WrittenBookEditor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_title">
        <property name="text">
         <string>标题</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_title">
        <property name="toolTip">
         <string>使用{volume}占位符可以在书名中添加卷号</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_export_options">
     <property name="title">
      <string>导出选项</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_item_type">
        <property name="text">
         <string>物品类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmb_item_type">
        <item>
         <property name="text">
          <string>成书</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>潜影盒</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_file_type">
        <property name="text">
         <string>文件类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmb_file_type">
        <item>
         <property name="text">
          <string>指令纯文本</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mc 函数文件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数据包</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_minecraft_version">
        <property name="text">
         <string>MC 版本</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmb_minecraft_version">
        <item>
         <property name="text">
          <string>&gt;=1.13 &lt;1.20.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=1.20.5 &lt;1.21</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=1.21</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_extra_options">
     <property name="toolTip">
      <string>如果你不知道这些配置的含义请不要随意修改</string>
     </property>
     <property name="title">
      <string>额外选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chk_text_component">
        <property name="text">
         <string>使用文本组件</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_filter">
        <property name="text">
         <string>使用过滤</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_0">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_export">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
